### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by ChGen**
Updated dependencies to the latest versions available.

### CODE REVIEW

**Comment by ChGen** [comment]
@olenagerasimova , please review.
I think some artipie dependencies may have newer versions in the git repository, but they are not available from the maven.

File: pom.xml [hunk 1]
```diff
@@ -45,17 +45,17 @@ SOFTWARE.
     <dependency>
       <groupId>com.artipie</groupId>
       <artifactId>asto</artifactId>
-      <version>1.2.1</version>
+      <version>v1.12.3</version>
```
Comment (olenagerasimova) [inline pom.xml:48]: please, use `asto-core` v1.13.0 instead of `asto`

**Review by olenagerasimova — DISMISSED** [review]
@ChGen just one suggestion, please, correct

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Review by olenagerasimova — APPROVED** [review]
thanks

**Pull request merged**